Containerized Docker Application Lifecycle - 06.4 (#10624)
dotnet/docs: docs/standard/containerized-lifecycle-architecture/run-manage-monitor-docker-environments/monitor-containerized-application-services.md

To create an Application Insights resource, follow the instructions in the articles presented in the list that follows. Azure portal will create the necessary script for you.

- **Monitor Docker applications in Application Insights:** \
  <https://docs.microsoft.com/azure/application-insights/app-insights-docker>

- **Application Insights Docker image at Docker Hub and GitHub:** \
  <https://hub.docker.com/r/microsoft/applicationinsights/> and \
  <https://github.com/Microsoft/ApplicationInsights-Docker>

- **Set up Application Insights for ASP.NET web apps and ASP.NET Core:** \
  <https://docs.microsoft.com/azure/application-insights/app-insights-asp-net>

- **Application Insights for web pages:**  
  <https://docs.microsoft.com/azure/application-insights/app-insights-javascript>

## Security, backup, and monitoring services

There are many support chores with lots of details that you have to handle to ensure your applications and infrastructure are in top notch condition to support business needs, and the situation gets heavier in the microservices realm, so you need a way to have both high-level and detailed views when you need to take action.

> Azure has the tools to manage both your cloud and on-premises resources in a unified view if four critical aspects:

Reviewer: ```suggestion
Azure has the tools to manage an provide a unified view of four critical aspects of both your cloud and on-premises resources:
```

Author: That makes an important difference!

- **Security**. With [Azure Security Center](https://azure.microsoft.com/services/security-center/).
  - Get full visibility and control over the security of your virtual machines, apps, and workloads.
  - Centralize the management of your security policies and integrate existing processes and tools.
  - Detect real threats with advanced analytics.

- **Backup**. With [Azure Backup](https://azure.microsoft.com/services/backup/).
  - Avoid costly business disruptions, meet compliance goals, and protect your data against ransomware and human errors.
  - Keep your backup data encrypted in transit and at rest.
  - Ensure access based on multifactor authentication to prevent unauthorized use.

- **Monitoring**. With [Azure monitoring](https://azure.microsoft.com/solutions/monitoring/), [Log Analytics](https://azure.microsoft.com/services/log-analytics/), and [Application Insights](https://azure.microsoft.com/services/application-insights/).
  - Get full visibility into the health and performance of your Azure workloads, apps, and infrastructure.

>   - Collect data from any source and get rich insights for your virtual machines, containers, and applications.

Reviewer: ```suggestion
  - Collect data from any source and get rich insights into your virtual machines, containers, and applications.
```
